kythe/typescript/testdata/class.ts

```diff
@@ -38,8 +40,10 @@ class Class implements IFace {
     //- ClassCtor.subkind constructor
     //- @otherMember defines/binding OtherMember
     //- OtherMember.node/kind variable
+    //- OtherMember childof Class
     //- @member defines/binding FakeMember
     //- FakeMember.node/kind variable
+    //- !{ FakeMember childof Class }
```

Comment:
FakeMember should also be childof the constructor.

Reply:
You're right. I'll resolve this.